src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -13927,6 +13927,20 @@ public abstract class MulticastDelegate : Delegate { }
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "2m").WithArguments("System.Decimal").WithLocation(2, 24));
         }
 
+        [Fact]
+        public void ParamsArray_MissingParamArrayAttribute()
+        {
+            var source = """
+                var lam = (params int[] xs) => xs.Length;
```

Comment:
>(params int[] xs) => xs.Length; [](http://example.com/codeflow?start=26&length=31)

Are there other, perhaps non-lambda scenarios, that are supposed to produce this error? There are a lot of code changes with the intent to report the error. Are all those code paths covered by this single test?

Reply:
You are right, thanks. Added more tests: 03aa9e599442324cb52e5156432b17652e051d9d